python_modules/dagster/dagster_tests/definitions_tests/test_external_assets.py

```diff
@@ -209,18 +209,14 @@ def an_asset(context: AssetExecutionContext, source_asset: str) -> str:
     assert result_one.output_for_node("an_asset") == "hardcoded-computed-2021-01-02"
 
     shimmed_source_asset = create_external_asset_from_source_asset(source_asset)
-    defs_with_shim = Definitions(
-        assets=[create_external_asset_from_source_asset(source_asset), an_asset]
-    )
+    defs_with_shim = Definitions(assets=[shimmed_source_asset, an_asset])
 
     assert isinstance(defs_with_shim.get_assets_def("source_asset"), AssetsDefinition)
 
-    job_def_with_shim = get_job_for_assets(defs_with_shim, an_asset, shimmed_source_asset)
+    job_def_with_shim = get_job_for_assets(defs_with_shim, an_asset)
```

Comment:
We no longer need to provide the external asset when getting the job that uses it purely for loading.